sentry-kotlin-multiplatform/src/androidMain/kotlin/io/sentry/kotlin/multiplatform/SentryAndroid.kt

```diff
@@ -1,7 +1,8 @@
 package io.sentry.kotlin.multiplatform
 
 import io.sentry.Sentry
-import io.sentry.android.core.SentryAndroidOptions
+
+private val globalScope = SentryScope()
```

Comment:
Why do we need a `globalScope`?

Reply:
This was my first iteration, now that we are directly modifying the scope we don't need it anymore